docs/tutorials/vision/large_scale_classification.md

```diff
@@ -0,0 +1,246 @@
+# Large Scale Image Classification
+
+Training a neural network with a large number of images presents several challenges. Even with the latest GPUs it is not possible to train large networks using large number of images in reasonable amount of time using a single GPU. This problem can be somewhat mitigated by using multiple GPUs in a single machine. But there is a limit to the number of GPUs that can be attached to one machine (typically 8 or 16). This tutorial explains how to train large networks with terabytes of data using multiple machines each containing multiple GPUs.

```

Comment:
I recommend that you run this narrative through grammarly.com. It will catch a bunch of minor issues that I might suggest.

For example: 
* comma after GPUs
* to train large number -> to train a large number
* images in reasonable amount of time -> images in a reasonable amount of time